webClient/src/app/shared/dialog/overwrite-dataset/overwrite-dataset.component.html

```diff
@@ -0,0 +1,30 @@
+
+<!-- 
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+-->
+<zlux-tab-trap></zlux-tab-trap>
+<h2 mat-dialog-title>Overwrite Dataset</h2>
```

Comment:
Somewhat personal preference, you can change if you think it makes sense to you: The title should be "E-tag Mismatch" Description should be "The dataset {{datasetName}} has been previously updated on the server. Would you like to overwrite the dataset?"